packages/server/graphql/public/types/CreateOneOnOneTeamInput.ts

```diff
@@ -0,0 +1,24 @@
+import {GraphQLInputObjectType, GraphQLNonNull, GraphQLString} from 'graphql'
+
+const CreateOneOnOneTeamInput = new GraphQLInputObjectType({
+  name: 'CreateOneOnOneTeamInput',
+  fields: () => ({
+    userId: {
```

Comment:
What's the use case where we'll provide a `userId` but not an `email`?

Reply:
I was thinking that when you select existing user in the user picker, we will provide a userId, however, I think I can simplify and just provide emails